spinalcordtoolbox/deepseg_gm/deepseg_gm.py

```diff
@@ -183,8 +227,8 @@ def segment_volume(ninput_volume, model_name):
     axial_slices = np.asarray(axial_slices, dtype=np.float32)
     axial_slices = np.expand_dims(axial_slices, axis=3)
 
-    axial_slices -= metadata['mean_train']
-    axial_slices /= metadata['std_train']
+    normalization = VolumeStandardizationTransform()
+    axial_slices = normalization(axial_slices)
```

Comment:
Use the more accurate term standardization (vs. normalization)?

Reply:
Standardization is a type of normalization.